guides/development-workflow.md

```diff
@@ -401,27 +414,87 @@ Upgrade widget to v1.2.1
+     any alternative ideas before making recommendations
+   - use the [following checklist](#what-to-look-for) to help assess the
+     contents
+1. submit your review and let the author know
+
+Tip: You can add `?w=1` to the end of the URL to strip our noisy whitespace.
+
+### What to look for
+
+When performing a code review we:
+
+- check the pull request is descriptive, explains the story and doesn't assume
+  knowledge. Some templates ask for screenshots to help illustrate the change
+  before and after
+- check the code is safe
+  ([OWASP top ten](https://owasp.org/www-project-top-ten/)) (Brakeman should be
+  checking for obvious issues automatically)
+- check the code follows a consistent style (Standard should be checking this
+  automatically)
+- check the code is readable (did you find it easy to build an understanding?)
+- check the code for maintainability
+  ([SOLID principles](https://www.digitalocean.com/community/conceptual_articles/s-o-l-i-d-the-first-five-principles-of-object-oriented-design))
+- check the code is tested (Test suites and Coverage tools should checked
+  automatically)
+- check the code is unlikely to introduce future bugs (this is tricky to do and
+  it's okay if you don't find any as we fail fast and fix)
+- check the commits are reasonably small and focused towards making one change.
+  They should not include unexpected or unrelated changes
+- check the commits messages are descriptive
+- check the [changelog](#tracking-changes) has been updated
+
+### Writing feedback
 
 It is not important that a story be implemented exactly how you would have done
-it. Only that it meets the criteria above. When we find problems, we give
-feedback to the developer who implemented the story, not fix things ourselves
-inline with our principle of ['Own your code'](#own-your-code).
+it, only that it meets the acceptance criteria. When we find problems we give
+feedback to the author rather than fix things ourselves inline with our
+principle of ['Own your code'](#own-your-code).
 
-It’s also important that code reviews have a constructive, amicable tone. To
-this end, we bear in mind the
-[Thoughtbot](https://github.com/thoughtbot/guides/tree/master/code-review) code
-review guide, which contains good rules for keeping things positive and useful.
+It’s important that code reviews have a constructive and amicable tone. A
+helpful mindset is one of assuming good intent. Be mindful that you might not
+have all the information and so ask questions such as "Why did you do this like
+X?" rather than "You should do it this like Y!".
+
+Summarise your feedback. Be clear on which points require further action and
+which are advisory. You should both feel confident about what's necessary and
+what to expect in order to move forward.
+
+[Thoughtbot's](https://github.com/thoughtbot/guides/tree/master/code-review)
+code review guide contains good rules for keeping the tone positive.
```

Comment:
Should we have a similar guide for how to respond to feedback on a PR? What do you do if you disagree with some feedback, for instance?

Reply:
Not necessary in this change, I think, btw.